src/app/profiles.rs

```diff
@@ -267,6 +274,12 @@ fn convert_route(
     Some((req_match, route))
 }
 
+fn convert_dst_override(orig: api::WeightedDst) -> Option<(NameAddr, u32)> {
+    NameAddr::from_str(orig.authority.as_str())
+        .ok()
+        .map(|addr| (addr, orig.weight))
+}
```

Comment:
I suggest that this (or another filter) drop any route with a 0-weight?

Or, rather, what is the expected behavior with zero-weights? What if all overrides have 0 weights? is that the same as having no weights at all?